src/internal/utils/mdObjectHelper.ts

```diff
@@ -73,3 +74,48 @@ export function canSortStackTotalValue(
         hasOneViewItem(mdObject)
     );
 }
+
+export function countItemsInMdObject(mdObject: VisualizationObject.IVisualizationObjectContent) {
+    const viewByItemCount: number = getBucketItemsByTypeFromMdObject(mdObject, BucketNames.VIEW).length;
+    const measureItemCount: number = getBucketItemsByTypeFromMdObject(mdObject, BucketNames.MEASURES).length;
+    const secondaryMeasureItemCount: number = getBucketItemsByTypeFromMdObject(
+        mdObject,
+        BucketNames.SECONDARY_MEASURES,
+    ).length;
+
+    return {
+        viewByItemCount,
+        measureItemCount,
+        secondaryMeasureItemCount,
+    };
+}
+
+export function countItemsOnAxes(
+    type: string,
+    controls: any,
```

Comment:
`any` :)

Reply:
`controls` is persisted as string at BE side, so I think `any` is ok

Reply:
Fixed